src/main/java/com/airsquared/blobsaver/Shared.java

```diff
@@ -88,30 +96,35 @@ protected Void call() throws Exception {
                         try {
                             response = makeRequest(new URL("https://api.github.com/repos/airsquared/blobsaver/releases/latest"));
                         } catch (FileNotFoundException ignored) {
-                            return null;
+                            newUnreportableError("Could not check for updates. " +
```

Comment:
I think a better idea would be to do `catch (Exception e) { ...` and remove the abandoned/moved elsewhere part because if this was the case, the repository would still be accessible and just archived.

Reply:
How about this?
```
catch (FileNotFoundException e) {
    System.err.println("https://api.github.com/repos/airsquared/blobsaver/releases/latest " +
            "doesn't exist??");
    e.printStackTrace();
    return null;
} catch (IOException e) {
    System.err.println("unknown IOException while checking for updates :( " +
            "Are you connected to the internet?");
    e.printStackTrace();
    return null;
}
```

Reply:
Another idea: do `catch (IOException e)` and show an alert to the user saying unable to check for updates.